aboutme.html

```diff
@@ -0,0 +1,102 @@
+                <span class="underline-style"></span>
+            </h2>
+
+            <p class="lato-font-black">
+                This year's event poster was chosen from the best of many after a tough competition between designers from different backgrounds. The selection was made for the creation of Brown Ariana.
+            </p>
+            <img src="./img/affiche.png" alt="Togoville Jazz Festival 2022">
+        </section>
+        <section class="past-festival">
+            <h2 class="lato-font-black">
+                See the past Togoville Jazz Festival
+                <span class="underline-style"></span>
+            </h2>
+
+            <p class="lato-font-black" >
+                Take a look at the last two Togoville Jazz Festival which took place at French Institute and Level area in Lomé 
+            </p>
+            <div class="last-festival-showed">
+                <div class="last-festival-div">
+                    <img class="last-festival" src="./img/276022750_2424200877717462_4783049319758784780_n.jpg" alt="Togoville Jazz Festival 2022">
+                    <div class="lato-font-black last-festival-description">
+                        <h5>2021</h5>
+                        <p>Togoville Jazz Festival 2021 at French institute</p>
+                    </div>
+                </div>
+                <div class="last-festival-div">
+                    <img class="before-last" src="./img/1024x538_cmsv2_2276f65d-03d1-5acc-b66c-20e9c085ffb6-6575712.jpg" alt="Togoville Jazz Festival 2021">
+                    <div class="lato-font-black before-last-decription">
+                        <h5>2020</h5>
+                        <p>Togoville Jazz Festival 2020 at Level area</p>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="partner-pool" id="sponsor">
+            <h3 class="lato-font-black">Partner<span class="underline-style"></span></h3>
+            <ul class="partner-logo">
+                <li><img class="partner" src="./img/logo/inst fr 2.jpg" alt="fanga music logo"></li>
+                <li><img class="partner" src="./img/logo/fangamusic.jpg" alt="fanga music logo"></li>
+                <li><img class="partner" src="./img/logo/goethe inst 2.jpg" alt="goet institute logo"></li>
+                <li><img class="partner" src="./img/logo/onomo 1.jpg" alt="onomo hotel logo"></li>
+                <li><img class="partner" src="./img/logo/cafe info 1.png" alt="cafe informatique logo"></li>
+                <li><img class="partner" src="./img/logo/moov on arts 1.png" alt="aldus logo"></li>
+            </ul>
+        </section>
+
+    </main>
+    <footer class="copyright">
+        <img src="./img/level production icon 3.png" alt="level production icon">
+        <div class="right-reseved">
+            <p class="lato-font-black">2022 Level Production.</p>
+            <p class="lato-font-black">Some Right Reserved.</p>
+        </div>
+    </footer>
```

Comment:
- [ ] According to the template the partner section is not displayed on the About page. The footer also has a dark background. Kindly implement it accordingly. 🧐️

**Your design**

![Screenshot from 2023-02-09 13-23-40](https://user-images.githubusercontent.com/17243325/217802210-98bdd0ec-2dbf-493d-9372-1f063d0de794.png)

**Template**

![Screenshot from 2023-02-09 13-22-59](https://user-images.githubusercontent.com/17243325/217802330-8c06210a-0b73-4828-8f36-5dd3862d4533.png)

Reply:
Hi @amejid 
Thank you for your insightful reviews. I have a couple of questions regarding the design of the mobile version of our website:

- Should the partner section be removed from the About Me page for the mobile version?
- Is it acceptable to maintain the current light color for the footer on the mobile version?

Reply:
Hi @Matthieu96Code, the partner section should be displayed on the About page for the mobile version. 
The footer should be light on the mobile version. The color variation is required for the desktop version only.